### PR TITLE
update some guide titles and descriptions

### PR DESCRIPTION
**Pull request by sryza**
### Summary & Motivation

My motivation for this was to make the difference between /guides/dagster/how-assets-relate-to-ops-and-graphs and /guides/dagster/enriching-with-software-defined-assets more clear.  I made a few other changes as well while there.

### CODE REVIEW

**Review by erinkcochran87 — APPROVED** [review]
Ty!

One other thing I'd check before you merge this is if the **Moving to SDAs** guide is referenced by title anywhere. Best way to look for this is probably to search for its file path (`/guides/dagster/enriching-with-software-defined-assets`) and looking at the display text.

**Comment by sryza** [comment]
> Before you merge this, I'd also check if the Moving to SDAs guide is referenced by title anywhere. Best way to look for this is probably to search for its file path (/guides/dagster/enriching-with-software-defined-assets) and looking at the display text.

Just did a search and didn't come across any

**Pull request merged**
